README.md

```diff
@@ -1,10 +1,10 @@
-# New Relic Integration
+# New Relic integration
```

Comment:
why the lower case here? Shouldn't section titles be title-cased with init caps?

Reply:
Our style (at least in docs and in most properties I know of) is sentence case, meaning that you don't capitalize something unless it's a proper name. If you want to google more about it, google 'sentence case vs title case'. We use sentence case. Not a big deal, as there are many teams whose stuff we haven't reviewed who use title case. But it's the preferred style.

Reply:
I actually wonder if this title should be "New Relic Kamon integration". Not clear why it's just 'NR integration' and not specific.